src/foundry/foundry.js/roll.d.ts

```diff
@@ -1,6 +1,4 @@
-// TODO: Remove when updating this class!!!
-// eslint-disable-next-line
-// @ts-nocheck
+import { ConfiguredDocumentClass } from '../../types/helperTypes';
```

Comment:
Since this is now an ESmodule once you use any `import` or `export` statements, you have to `export` the `Roll` class to make it accessible.

Reply:
Simply `export declare class …` does not resolve the issue. Wrapping it with `declare global` (and removing the inner `declare`) does, but I'm not sure whether this is the right direction.

Reply:
`Roll` is defined in foundry.js, so it is available globally (as foundry.js you s a script and not a module). If we make `roll.d.ts` a module because we need to import stuff, using `declare global` to still make `Roll` global is indeed the right thing to do.